### PR TITLE
Remove duplicated code

### PR DESCRIPTION
**Pull request by reneeb**
## Proposed change

The DynamicField modules were declared as dependencies twice and the VariableCheck functions were imported twice.
This change is just for the code cleanliness.

## Type of change
- 🆙 - Dependency upgrade (e.g. libraries)
- 🐞 - Bugfix (non-breaking change which fixes an issue)
- [X] 💎 - Code quality improvements to existing code or addition of unit tests
- 🚀 - New feature (which adds functionality to an existing integration)

## Checklist

- [X] The code change is tested and works locally.(❗)
- [ ] There is no commented out code in this PR.(❕)
- [ ] You improved or added new unit tests.(❕/❗depends on the code)
- [ ] Local ZnunyCodePolicy run passes successfully.(❗)
- [X] Local unit tests pass.(❗)
- [x] GitHub workflow ZnunyCodePolicy passes.(❗)
- [x] GitHub workflow unit tests pass.(❗)

### CODE REVIEW

File: Kernel/System/CustomerUser/DB.pm [hunk 1]
```diff
@@ -16,8 +16,6 @@ use Kernel::System::VariableCheck qw(:all);
 use Crypt::PasswdMD5 qw(unix_md5_crypt apache_md5_crypt);
 use Digest::SHA;
 
-use Kernel::System::VariableCheck qw(:all);
```
Comment (dennykorsukewitz) [inline Kernel/System/CustomerUser/DB.pm:19]: @reneeb IsArrayRefWithData and IsHashRefWithData are used in the code.

**Review by dennykorsukewitz — COMMENTED** [review]
Hi @reneeb please check `use Kernel::System::VariableCheck qw(:all);`

File: Kernel/System/CustomerUser/DB.pm (hunk 1 above)
Comment (reneeb) [reply Kernel/System/CustomerUser/DB.pm:19]: That's why the "use" in line 14 is stll there ;-)

**Review by dennykorsukewitz — APPROVED** [review]
👍🏼

**Pull request merged**